<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>801</width>
      <height>611</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777212</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame_2{
background-color:rgba(12, 30, 41, 240);
}


</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>DSA LAB</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(215, 215, 215)
</string>
     </property>
     <property name="text">
      <string>Flight Name</string>
     </property>
     <property name="placeholderText">
      <string>Imported File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RegEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(215, 215, 215)
</string>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
     <property name="placeholderText">
      <string>Imported File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gpaEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(215, 215, 215)
</string>
     </property>
     <property name="text">
      <string>Ticket Price</string>
     </property>
     <property name="placeholderText">
      <string>Imported File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="uniEdit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(215, 215, 215)
</string>
     </property>
     <property name="text">
      <string>Destination</string>
     </property>
     <property name="placeholderText">
      <string>Imported File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color:rgb(17, 30, 42);
	border: none;
   color: white;
    border-radius:10px;
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: white;
   color : black
}</string>
     </property>
     <property name="text">
      <string>Add Flight</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EditButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>120</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color:rgb(17, 30, 42);
	border: none;
   color: white;
    border-radius:10px;
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: white;
   color : black
}</string>
     </property>
     <property name="text">
      <string>Edit Flight</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>160</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color:rgb(17, 30, 42);
	border: none;
   color: white;
    border-radius:10px;
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: white;
   color : black
}</string>
     </property>
     <property name="text">
      <string>Delete Flight</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="StuInfoTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>731</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {

background-color:rgb(215, 215, 215);
color:black;


}</string>
     </property>
     <column>
      <property name="text">
       <string>Flight Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ticket Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Destination</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="CrossButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: none;
	background-color: rgba(0,0,0,0);
   color: white
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: rgba(0,0,0,0);
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
